components/grid/demo/flex-align.md

```diff
@@ -16,7 +16,7 @@ Flex child elements vertically aligned.
 ```jsx
 import { Row, Col } from 'antd';
 
-const DemoBox = props => <p className={`height-${props.value}`}>{props.children}</p>;
+const DemoBox = props => <p style={{ height: props.value }}>{props.children}</p>;
```

Comment:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/507615/62679297-aa29e600-b9e6-11e9-9445-1bf5d555ecbb.png">

There is `line-height` style to cover.